assembl/static2/js/app/components/cookies/cookieSetter.jsx

```diff
@@ -0,0 +1,72 @@
+// @flow
+/* global globalAnalytics */
```

Comment:
what is this?

Reply:
It's a global variable needed to access the matomo_host of the server

Reply:
I know globalAnalytics is used in V1 interface, in fact, for piwik. We use React-Router-Piwik in V2, which is completely different, and this variable becomes redundant.

Reply:
Is this the same thing? This was added by @vincentfretin on analytics_common.jinja2 on the previous cookies branch